src/main/java/com/google/summit/ast/declaration/ClassDeclaration.kt

```diff
@@ -34,14 +35,14 @@ class ClassDeclaration(
   id: Identifier,
   val extendsType: TypeRef?,
   val implementsTypes: List<TypeRef>,
-  bodyDeclarations: List<Declaration>,
+  bodyDeclarations: List<Modifiable>,
```

Comment:
Class declaration members are not all `Declaration`s now; the most specific type they can be is `Modifiable`.

Should this be a `List<Modifiable>` or more general `List<Node>`?

Should `bodyDeclarations` be renamed to `members`? Though `FieldDeclarationGroup` isn't a `Declaration`, it still fits the more broad label of "declaration" in my opinion, so maybe the current name is fine.

Reply:
> Should this be a List<Modifiable> or more general List<Node>?
IMHO, the more relevant aspect is that these are Nodes and not that they are Modifiable objects.

> it still fits the more broad label of "declaration" in my opinion, so maybe the current name is fine.
Agreed. It contains declarations.

Reply:
> IMHO, the more relevant aspect is that these are Nodes and not that they are Modifiable objects.

I forgot that `Modifiable` isn't also a `Node`. I'll change this to `List<Node>` and a few more instances of `Modifiable` to `Node` in `Translate.kt`.